app/main/views/services.py

```diff
@@ -150,6 +150,8 @@ def update_section(service_id, section_id):
             errors=errors
         )
 
+    flash(u"You\u2019ve edited your service. The changes are now live on the Digital Marketplace.", 'message')
```

Comment:
Only reason I did it this way is it's currently the only way we have of delivering this in a non-destination-page-specific (and hence robust) way. And there's no way it can accidentally be misinterpreted by the wrong template and dumped to the screen as a raw dictionary as is possible in some cases.